templates/items/parts/item-description.hbs

```diff
@@ -51,6 +51,26 @@
         {{/if}}
     </div>
 
-    {{editor descriptionHTML target="system.description.value" button=true editable=editable engine="prosemirror"
-             collaborate=false}}
+    <div class="flexcol">
+        {{#unless isIdentifiable}}
+            {{editor descriptionHTML target="system.description.value"
+                button=true editable=editable engine="prosemirror" collaborate=false}}
+        {{/unless}}
+        {{#if showIdentifiedDesc}}
+            {{editor descriptionHTML target="system.description.value"
+                button=true editable=editable engine="prosemirror" collaborate=false}}
+        {{/if}}
+        {{#if showUnidentifiedDesc}}
+            {{#if isGM}}
+                <hr class="flex0">
```

Comment:
Is this `<hr>` just for spacing? It doesn't seem to show up visually. If so, it would probably be better to do with with CSS on the unidentified description box.

Reply:
I think this `<hr>` was a remnent from showing a line that I removed, it will be gone.